packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -75,6 +101,76 @@ export class Graph extends PureComponent<GraphProps> {
     );
   }
 
+  renderTooltip = () => {
+    const { children, series, timeZone } = this.props;
+    const { pos, activeItem, isTooltipVisible } = this.state;
+    const tooltips: React.ReactNode[] = [];
+
+    if (!isTooltipVisible || !pos || series.length === 0) {
+      return null;
+    }
+
+    // Find children that indicate tooltip to be rendered
+    React.Children.forEach(children, c => {
+      // @ts-ignore
+      const childType = c && c.type && (c.type.displayName || c.type.name);
+
+      if (childType === Tooltip.displayName) {
+        tooltips.push(c);
+      }
+    });
```

Comment:
Could this be optimized to break when first child is found that satisfies condition?